tests/unit_tests/gui/test_gui_load.py

```diff
@@ -132,6 +132,10 @@ def test_that_loading_gui_creates_a_single_storage_folder(
     monkeypatch.setattr(ert.gui.gert_main, "QApplication", Mock(return_value=qapp))
     monkeypatch.setattr(ert.gui.gert_main.LibresFacade, "enspath", tmp_path)
     run_gui(args)
+    lists = [glob for glob in tmp_path.glob("**/*") if "storage" in glob.__str__()]
+    for fil in lists:
```

Comment:
Is this leftover debug code_

Reply:
yeah it was, sorry, it's gone in the end